### PR TITLE
Email fix

### PR DESCRIPTION
**Pull request by SchrodingersGat**
Fixes issues with confirming email addresses

Closes https://github.com/inventree/InvenTree/issues/5333
Reference https://django-rest-auth.readthedocs.io/en/latest/faq.html

### CODE REVIEW

**Comment by SchrodingersGat** [comment]
@matmair I've worked out why the endpoint did not have a template - but, the LOGIN_REDIRECT_URL is now hardcoded for the "platform" UI - this seems like an oversight, as this is not anywhere near being the default UI yet?

File: InvenTree/InvenTree/settings.py [hunk 1]
```diff
@@ -608,7 +608,7 @@
 
 # Magic login django-sesame
 SESAME_MAX_AGE = 300
-LOGIN_REDIRECT_URL = "/platform/logged-in/"
+LOGIN_REDIRECT_URL = "/logged-in/"
```
Comment (matmair) [inline InvenTree/InvenTree/settings.py:611]: @SchrodingersGat pretty sure this endpoint does not exist - `/plattform/logged-in/` is a jsx endpoint on P UI, not a Django endpoint

File: InvenTree/InvenTree/urls.py [hunk 2]
```diff
@@ -79,13 +80,16 @@
...
-    path('auth/social/<int:pk>/disconnect/', SocialAccountDisconnectView.as_view(), name='social_account_disconnect'),
+    # Auth API endpoints
+    path('auth/', include([
+        re_path(r'^registration/account-confirm-email/(?P<key>[-:\w]+)/$', ConfirmEmailView.as_view(), name='account_confirm_email'),
+        path('registration/', include('dj_rest_auth.registration.urls')),
+        path('providers/', SocialProvierListView.as_view(), name='social_providers'),
+        path('social/', include(social_auth_urlpatterns)),
+        path('social/', SocialAccountListView.as_view(), name='social_account_list'),
+        path('social/<int:pk>/disconnect/', SocialAccountDisconnectView.as_view(), name='social_account_disconnect'),
+        path('', include('dj_rest_auth.urls')),
```
Comment (matmair) [inline InvenTree/InvenTree/urls.py:91]: This should be the first item so we can overwrite several defaults from dj-rest-auth

**Comment by matmair** [comment]
@SchrodingersGat LOGIN_REDIRECT_URL is needed for magic login so we either need to cut that feature (https://github.com/inventree/InvenTree/issues/3531) from 0.13 or write some view that redirects the user and selects somehow which UI the users currently uses.

File: InvenTree/InvenTree/urls.py (hunk 2 above)
Comment (SchrodingersGat) [reply InvenTree/InvenTree/urls.py:91]: @matmair sorry which line in particular should be first?

**Comment by SchrodingersGat** [comment]
> @SchrodingersGat LOGIN_REDIRECT_URL is needed for magic login so we either need to cut that feature (#3531) from 0.13 or write some view that redirects the user and selects somehow which UI the users currently uses.

For now, the solution needs to ensure that the current (legacy) UI is hard-coded. A future PR can be added to support platform too, but current master assumes the login redirect will go to "platform" which is not correct

**Pull request merged**